docs/source/commandline/commandline-all.rst

```diff
@@ -107,54 +144,76 @@ Note: By default, all the '**-list**' commands below write info to the screen. I
 		* 8: means some files were identified
 		* 9: means no files were identified
 
+.. _mame-commandline-listdevices:
+
 **-listdevices** / **-ld** [<*gamename|wildcard*>]
 
         Displays a list of all devices known to be hooked up to a game.  The ":" is considered the game itself with the devices list being attached to give the user a better understanding of what the emulation is using.
 
+.. _mame-commandline-listslots:
+
 **-listslots** [<*gamename|wildcard*>]
 
         Show available slots and options for each slot (if available).  Primarily used for MAME to allow control over internal plug-in cards, much like PCs needing video, sound and other expansion cards.
 		
         The slot name (e.g. **ctrl1**) can be used from the command line (**-ctrl1** in this case) 
 
+.. _mame-commandline-listmedia:
+
 **-listmedia** / **-lm** [<*gamename|wildcard*>]
 
         List available media that the chosen game or system allows to be used.  This includes media types (cartridge, cassette, diskette and more) as well as common file extentions which are supported.
 
+.. _mame-commandline-listsoftware:
+
 **-listsoftware** [<*gamename|wildcard*>]
 
         Posts to screen all software lists which can be used by the entered gamename or system.  Note that this is simply a copy/paste of the .XML file which reside in the HASH folder which are allowed to be used.
 
+.. _mame-commandline-verifysoftware:
+
 **-verifysoftware** [<*gamename|wildcard*>]
 
 	Checks for invalid or missing ROM images in your software lists. By default all drivers that have valid ZIP files or directories in the rompath are verified; however, you can limit this list by specifying a specific driver name or wildcard after the -verifysoftware command.
 
+.. _mame-commandline-getsoftlist:
+
 **-getsoftlist** [<*gamename|wildcard*>]
 
         Posts to screen a specific software list which matches with the gamename provided.
 
+.. _mame-commandline-verifysoftlist:
+
 **-verifysoftlist** [softwarelistname]
 
 	Checks a specified software list for missing ROM images if files exist for issued softwarelistname. By default, all drivers that have valid ZIP files or directories in the rompath are verified; however, you can limit this list by specifying a specific softwarelistname (without .XML) after the -verifysoftlist command.
 
 
 .. _osd-commandline-options:
 
-OSD related options
+OSD-related Options
 -------------------
 
+.. _mame-commandline-uimodekey:
+
```

Comment:
You probably want to note that the default is forward delete on Mac (fn-delete on notebook/compact keyboards, dedicated key on full-size keyboards).  It might be different again on Linux, I can't remember.

Reply:
Fixed on my side.